<launch>
    <node name="rover"         pkg="hummingbot" type="rover.py"/>
    <node name="motor_control" pkg="hummingbot" type="motor_control.py"/>

    <param name="velocity_coef" value="255.0" type="double"/>
    <!--dynamic topics-->
    <param name="velocity_topic_left"  value="/gazebo_hummingbot/left_vel"  type="string"/>
    <param name="velocity_topic_right" value="/gazebo_hummingbot/right_vel" type="string"/>
    
</launch>
